test/mason/build/noDeps.good

```diff
@@ -1,4 +1,3 @@
-Skipping registry update since no dependency found in manifest file.
```

Comment:
Why is the output gone? That feels like a bigger change?

Reply:
> I know I still owe an issue to discuss how/what Mason outputs

I'm still not totally sure I agree that this PR won't mean we aren't testing what users see, but I forgot about this. I think as a temporary measure this is fine but I wouldn't want to see it last long term.